### PR TITLE
Feature/richard branch

### PR DESCRIPTION
**Pull request by richardverheyen**
The sexiest goddamn mobile nav bar you've ever seen

### CODE REVIEW

File: src/styles/_header.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@ header {
   background: url('/assets/img/header/red-header-background.jpg');
   background-size: cover;
 
-  div {
+  > div {
```
Comment (janwerkhoven) [inline src/styles/_header.scss:5]: smart move

Comment (richardverheyen) [reply src/styles/_header.scss:5]: Picking that little guy out saved me like 3 conflicts between the logo and my dropdown images

File: src/styles/_common.scss [hunk 2]
```diff
@@ -122,27 +128,70 @@ button {
...
+    }
+
+    img:nth-last-child(1) {
+      display: block;
+    }
+  }
+}
+//Header Mobile Navigation Toggle
+
+.mobileNavActive {
```
Comment (janwerkhoven) [inline src/styles/_common.scss:154]: no camelCasing in CSS please, in CSS all names are dasherised

File: src/styles/_header.scss (hunk 1 above)
Comment (janwerkhoven) [reply src/styles/_header.scss:5]: Yea best to be precise like this

File: src/styles/_common.scss (hunk 2 above)
Comment (janwerkhoven) [reply src/styles/_common.scss:154]: I'll accept it, but try not to in future

**Comment by janwerkhoven** [comment]
LGTM ⛵️

**Pull request merged**